src/include/distributed/master_protocol.h

```diff
@@ -103,7 +103,7 @@ extern bool IsCoordinator(void);
 extern bool CStoreTable(Oid relationId);
 extern uint64 GetNextShardId(void);
 extern uint64 GetNextPlacementId(void);
-extern Oid ResolveRelationId(text *relationName);
+extern Oid ResolveRelationId(text *relationName, bool failOK);
```

Comment:
can we use `missingOK` instead ?

Reply:
sure, it's `missingOk` in the definition of the function so I'm not sure how I made this here 😄